tests/e2e/corrupt_test.go

```diff
@@ -12,11 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !cluster_proxy
+// +build !cluster_proxy
```

Comment:
There is no such build tag in https://github.com/etcd-io/etcd/pull/15924.

Reply:
Build tag change comes from https://github.com/etcd-io/etcd/pull/16637/commits/5b8150a6cfbe6e61c21f0848033a324c52895ecc commit, which also backport the test fix needed after https://github.com/etcd-io/etcd/pull/16677 changed the test.